### ISSUE TITLE
Add Sun-synchronous orbits

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#18]
First I need to add J2 to bodies, and then a function with this signature:

```
def sunsync(cls, attractor, alt, omega, ecc=0.0 * u.one, ...)
```

Actually if I want to define the local time at the ascending or descending node I need some kind of ephemerides. But for the moment I might just want to compute the inclination.

The point is that I might want to combine sso, frozen and repeated groundtrack for example. How can I do that?

**Comment by Juanlu001** [comment]
Blocked on #23.

**Comment by Juanlu001** [comment]
Out of scope, closing.
